omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/SystemSettingResource1_9.java

```diff
@@ -11,9 +11,6 @@
 
 import java.util.List;
 
-import io.swagger.models.Model;
-import io.swagger.models.ModelImpl;
-import io.swagger.models.properties.StringProperty;
```

Comment:
By changing the setting of the imports you make someone who is viewing the code wondering why this is so red. I guess it's your IDE's doing?